charts/gardener-extension-provider-aws/templates/daemonset-custom-mtu.yaml

```diff
@@ -28,12 +28,21 @@ spec:
       initContainers:
       - name: mtu-customizer
         image: {{ include "image.alpine" . }}
-        command: ["ip", "link", "set", "dev", "eth0", "mtu", "1460"]
+        command:
+        - /mtu-customizer/mtu-customizer.sh
         securityContext:
           capabilities:
             add: ["NET_ADMIN", "SYS_TIME"]
+        volumeMounts:
```

Comment:
Can you please add an annotation checksum to the annotations of this pod sec?

Reply:
done